quinn/src/endpoint.rs

```diff
@@ -356,12 +359,57 @@ pub(crate) struct EndpointInner {
     pub(crate) shared: Shared,
 }
 
+impl EndpointInner {
+    pub(crate) fn accept(
+        &self,
+        incoming: proto::Incoming,
+        mut response_buffer: BytesMut,
+    ) -> Result<Connecting, ConnectionError> {
+        let mut state = self.state.lock().unwrap();
+        state
+            .inner
+            .accept(incoming, Instant::now(), &mut response_buffer)
+            .map(|(handle, conn)| {
```

Comment:
Style nit: I prefer to avoid `map().map_err()` constructions in favor of just having a `match` expression, which uses more explicit syntax to do the same thing.

Reply:
Done.